Projects/Claims/Sources/SubmitClaimStore.swift

```diff
@@ -112,6 +112,15 @@ public final class SubmitClaimStore: LoadingStateStore<SubmitClaimsState, Submit
 
                 return disposeBag
             }
+        case let .submitTextInput(text):
```

Comment:
Do we really need a new action (.submitTextInput) here or can we reuse .submitAudioRecording in any way since we are calling the same mutation? Maybe send in a boolean argument as writtenText?

Reply:
Yeah, maybe we can do it with enum. Give me a sec